src/protagonist/DLCS.Repository/Migrations/DlcsContextModelSnapshot.cs

```diff
@@ -214,7 +214,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.HasIndex(new[] { "Customer", "Superseded", "Submitted" }, "IX_BatchTest");
 
-                    b.ToTable("Batches");
+                    b.ToTable("Batches", (string)null);

```

Comment:
Any idea why this `ToTable` call was updated to pass null schema?

Reply:
this was auto generated, so I'm not sure - don't think I modified anything to do with `Batches`